modules/RTC/TraceablePeerConnection.js

```diff
@@ -1392,8 +1392,8 @@ TraceablePeerConnection.prototype._addStream = function(mediaStream) {
  * @param {MediaStream} mediaStream
  */
 TraceablePeerConnection.prototype._removeStream = function(mediaStream) {
-    if (browser.isFirefox()) {
-        this._handleFirefoxRemoveStream(mediaStream);
+    if (browser.supportsRtpSender()) {
```

Comment:
thanks for renaming these, much better

Reply:
There's also a function which is more related to sdp-interloop caching for unified plan than to Firefox, I'll rename that too.